### PR TITLE
Getting relaced reduces your combat skills to nothing/basic for 30 minutes

### PR DESCRIPTION
**Pull request by Jackofashepard**
As the title says, your combat skills will be reduced to basic or nothing for 30 minutes. Dying is bad kids!

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
Absolutely fantastic. Prevents clones from trying to drown you in bodies, and makes headgibbing less essential.

File: code/modules/organs/internal/stack.dm [hunk 1]
```diff
@@ -69,7 +74,11 @@
 			overwrite()
 	sleep(-1)
 	do_backup()
-
+	to_chat(owner, "<span class='danger'>You feel sluggish and traumatized, best avoid anything violent for a while</span>")
+	owner.buff_skill(skilldecay, 1 HOUR, buff_type)//Debuff applied
+	ralacetime = world.time
```
Comment (Roland410) [inline code/modules/organs/internal/stack.dm:79]: Relacetime

File: code/modules/organs/internal/stack.dm [hunk 2]
```diff
@@ -69,7 +74,11 @@
 			overwrite()
 	sleep(-1)
 	do_backup()
-
+	to_chat(owner, "<span class='danger'>You feel sluggish and traumatized, best avoid anything violent for a while</span>")
```
Comment (Roland410) [inline code/modules/organs/internal/stack.dm]: Full stop at the end of sentence.

File: code/modules/organs/internal/stack.dm [hunk 3]
```diff
@@ -69,7 +74,11 @@
...
+	owner.buff_skill(skilldecay, 1 HOUR, buff_type)//Debuff applied
+	ralacetime = world.time
+	if(relacetime >= world.time + 1 HOUR)
+		to_chat(owner, "<span class='notice'>You feel like you have recovered slightly from your ordeal, still wouldn't make a habit of dying</span>")
```
Comment (Roland410) [inline code/modules/organs/internal/stack.dm:81]: Full stop.

**Comment by Jackofashepard** [comment]
It applies debuffs to Shooting, CQC and athletics which is reverted in 30 minutes (this was tested) the flavour message for recovering was added later

File: code/modules/organs/internal/stack.dm [hunk 4]
```diff
@@ -69,7 +74,11 @@
 			overwrite()
 	sleep(-1)
 	do_backup()
-
+	to_chat(owner,SPAN_WARNING("You feel sluggish and your limbs are heavy as your new body adjusts to the neural lace - you'll probably be pretty useless until your lace has acclimated."))
+	owner.buff_skill(skilldecay, 30 MINUTES, buff_type)//Debuff applied
+	relacetime = world.time
+	if(world.time == relacetime + 30 MINUTES)
+		to_chat(owner, "<span class='notice'>You feel like you have recovered slightly from your ordeal, still wouldn't make a habit of dying.</span>")
```
Comment (Roland410) [inline code/modules/organs/internal/stack.dm:81]: This.

**Comment by Superhats** [comment]
I personally think this should go through some level of test merge, considering how big of a change it is.

**Pull request merged**